src/test/resources/ant/build-simple.xml

```diff
@@ -0,0 +1,13 @@
+<project name="MyProject" default="testng" basedir=".">
+
+    <taskdef name="testng" classname="org.testng.TestNGAntTask" classpath="../../../../build/classes/java/main" />
```

Comment:
Gradle dependent.
I think it should be possible to inject gradle buildDir somewhere but I didn't dig.